EndlessSkyLib.cbp

```diff
@@ -330,13 +330,18 @@
 		<Unit filename="source/UI.h" />
 		<Unit filename="source/UniverseObjects.cpp" />
 		<Unit filename="source/UniverseObjects.h" />
+		<Unit filename="source/Variant.cpp" />
+		<Unit filename="source/Variant.h" />
 		<Unit filename="source/Visual.cpp" />
 		<Unit filename="source/Visual.h" />
 		<Unit filename="source/Weapon.cpp" />
 		<Unit filename="source/Weapon.h" />
 		<Unit filename="source/Weather.cpp" />
 		<Unit filename="source/Weather.h" />
 		<Unit filename="source/WeightedList.h" />
+		<Unit filename="source/WeightedVariant.cpp" />
+		<Unit filename="source/WeightedVariant.h" />
+		<Unit filename="source/gl_header.h" />
```

Comment:
Where'd this come from?

Reply:
You made a mistake while merging I think. `gl_header.h` was the old name for `opengl.h`.